src/EditorFeatures/Core/Shared/Extensions/HostWorkspaceServicesExtensions.cs

```diff
@@ -61,8 +61,8 @@ private static string GetDefaultContentTypeName(HostWorkspaceServices workspaceS
 
             if (mefHostServices != null)
             {
-                var registry = mefHostServices.GetExports<IContentTypeRegistryService>().Single().Value;
-                return mefHostServices.GetExports<ILanguageService, ContentTypeLanguageMetadata>()
+                var exports = mefHostServices.GetExports<ILanguageService, ContentTypeLanguageMetadata>();
```

Comment:
Why was this change needed? You wouldn't want to have unrelated language services impacting the result of this query I think.